modules/cloud-run-v2/variables.tf

```diff
@@ -229,19 +213,22 @@ variable "traffic" {
 variable "volumes" {
   description = "Named volumes in containers in name => attributes format."
   type = map(object({
-    secret_name  = string
-    default_mode = optional(string)
-    items = optional(map(object({
-      path = string
-      mode = optional(string)
-    })))
+    secret = optional(object({
+      secret       = string
```

Comment:
can we have `name` here, so then we can access it as `volumes[...].secret.name` ?

Reads better than `(...).secret.secret`

Reply:
I know, but I didn't want to deviate from the API naming to not confuse anyone. What about secret_name?

Reply:
I think we anyway deviate here a bit from provider attribute. I tend towards approach that `secret_name` is understood the same as `secret.name`, so I do not see that this might confuse (and this matches the [Secret Manager API](https://cloud.google.com/secret-manager/docs/reference/rest/v1beta1/projects.secrets#Secret)), though [terraform provider](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/secret_manager_secret_version) uses `secret_id` and `secret` :facepalm: 

Maybe its just me, who  is confused by that :smile: